lib/pf/Authentication/Source/AdminProxySource.pm

```diff
@@ -0,0 +1,153 @@
+package pf::Authentication::Source::AdminProxySource;
```

Comment:
I might be wrong, but I don't see anything preventing this source from being put on a portal profile by mistake

If its only for use in the admin, it should be removed from the portal profiles sources choices